### ISSUE TITLE
Fixed Potential Calcs with PC fluid-solvent

### ISSUE THREAD

**Issue by alex-aziz** [shankar1729/jdftx#240]
I have been running fixed potential calculations following the methodology in the tutorial. 

I have no issues when I use H2O as the solvent but when I change to the PC solvent, the calculations crash.

Alternatively, I can use H2O and reduce the dielectric to 65 to represent PC.

[fluid_PC.zip](https://github.com/shankar1729/jdftx/files/9247206/fluid_PC.zip)

Regards
Alex

**Comment by shankar1729** [comment]
Duplicate of #235

**Comment by shankar1729** [comment]
Fixed by https://github.com/shankar1729/jdftx/commit/f7da6bbb4f294c68b10a5d7011cf013ff0d2e4ba (not allowing unparameterized solvents for SaLSA and CANDLE, as they need more information than just the dielectric constant).

The solution is to use LinearPCM with the default (GLSSA13) variant, for which many other solvents are parameterized. Using CANDLE for H2O with a reduced dielectric constant is not a good idea, as you are then inheriting the charge asymmetry of H2O (which is the main point of CANDLE).

Also, I noticed in your input file that you are using 40 Eh cutoff for the GBRV pseudopotentials, which seems much higher than needed. Just checking to make sure this is not a Ry vs Eh mix up (QE and the GBRV paper use Ry, while JDFTx uses Eh units).

Best,
Shankar
